crates/algod_client/src/models/signed_txn_in_block.rs

```diff
@@ -0,0 +1,96 @@
+/*
+ * Algod REST API.
+ *
+ * API endpoint for algod operations.
+ *
+ * The version of the OpenAPI document: 0.0.1
+ * Contact: contact@algorand.com
+ * Generated by: Rust OpenAPI Generator
+ */
+
+use crate::models;
+#[cfg(not(feature = "ffi_uniffi"))]
+use algokit_transact::SignedTransaction as AlgokitSignedTransaction;
+use serde::{Deserialize, Serialize};
+use serde_with::{Bytes, serde_as};
+
+#[cfg(feature = "ffi_uniffi")]
+use algokit_transact_ffi::SignedTransaction as AlgokitSignedTransaction;
+
+use algokit_transact::AlgorandMsgpack;
+
+use crate::models::UnknownJsonValue;
+
+/// SignedTxnInBlock is a SignedTransaction with additional ApplyData and block-specific metadata.
+#[serde_as]
+#[derive(Clone, Default, Debug, PartialEq, Serialize, Deserialize)]
+#[cfg_attr(feature = "ffi_uniffi", derive(uniffi::Record))]
+pub struct SignedTxnInBlock {
```

Comment:
Can you elaborate on why we need a completely new struct and not just add a model that wraps the existing AlgokitSignedTransaciton? Something similar to `SignedTxnWithAD` in go-algorand:

```go
type SignedTxnWithAD struct {
	_struct struct{} `codec:",omitempty,omitemptyarray"`

	SignedTxn
	ApplyData
}
```